public/data/ihtishamKhan.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "Ihtisham",
+  "type": "personal",
+  "bio": "MERN Stack enthusiast",
+  "avatar": "https://github.com/ihtishamKhan.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/ihtishamKhan",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/ihtisham033",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ihtisham-khan/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Worked on college project",
+      "date": "2019-2020",
+      "icon": "graduation-cap",
```

`graduation-cap` is an incorrect keyword, use `graduation-hat` instead! 

```suggestion
      "icon": "graduation-hat",
```